app/views/layouts/quick_search/_search_form.html.erb

```diff
@@ -1,18 +1,26 @@
 <div class="search-container">
-  <h1>Search</h1>
-  <div class="main-search-form">
-    <%= form_for("search", method: :get, html: {id: 'main-search-form', class: 'form-inline'}) do |f| %>
-    <% if @query  %>
-    <input type="search" name="q" id="params-q" class="search-query form-control" value="<%= @query %>"/>
-  <% else %>
-    <input type="search" name="q" id="params-q" class="search-query form-control" placeholder="<%= @search_form_placeholder %>"/>
-    <% end %>
-    <span class="input-group-btn">
-      <button type="submit" class="btn search-btn">
-        <span class="sr-only">Search</span>
-        <i class="fa fa-search" aria-hidden="true"></i>
-      </button>
-    </span>
-    <% end %>
+  <div class="main-search-form row justify-content-center">
+    <div class="form-group col-8 text-center">
+      <%= form_for("search", method: :get, html: {id: 'main-search-form'}) do |f| %>
+        <div class='form-group row'>
+          <label for="params-q" class='col-md-4 col-form-label'>Search</label>
+          <div class='col-md-8 search-input-container'>
+            <div class='input-group'>
+              <% if @query  %>
+                <input type="search" name="q" id="params-q" class="search-query form-control" value="<%= @query %>"/>
```

Comment:
I know this is code you're just moving/cleaning up, but just wondering if this should just be the same element (w/ the `placeholder` and the `value`).  It would have the benefit of getting the only logic out of this partial.  Up to you though since you did not introduce this.

Reply:
Good catch.. I 🔥  the logic.